### PR TITLE
[Python] Gradient Descent

### PR DESCRIPTION
**Pull request by premalrupnur**
Implemented Gradient Descent using numpy , used sklearn for creating regression datasets

### CODE REVIEW

**Review by amanmehara — APPROVED** [review]
Thanks for Contributing! 🎉
